src/zjs_callbacks.c

```diff
@@ -86,6 +92,8 @@ void zjs_init_callbacks(void)
         }
         memset(cb_map, 0, size);
     }
+    zjs_port_ring_buf_init(&ring_buffer, 1024, (uint32_t*)args_buffer);
```

Comment:
James answered that it was to simplify implementing the linux port. We should look into whether it changes things or not, to make sure we're getting the good performance, but we could save that task for later. Also, this accidentally changed the buffer from 4096 bytes to 1024 bytes vs. the first version of the patch, but maybe that's enough. We'll need to profile to get a sense of a good default at some point.